user/multi-cpu-architectures.md

```diff
@@ -77,5 +78,36 @@ matrix:
    - os: linux
      env: LIB_PATH="/usr/bin/shared/x86_64/v2"
 ```
+{: data-file=".travis.yml"}
 
 For example, the above `.travis.yml`, would result in running both jobs with the environmental variable LIB_PATH assigned different values being run only on `amd64` architecture.
+
+## Using docker in `Arm` based builds within LXD container
+
+It is possible to use Docker in `Arm` based builds within a LXD container. You may need an arm64v8 docker image as a base or ensure arm64 libraries required by your build are added to your Dockerfile. In order to use docker in `Arm` based build within LXD container, docker commands must be run with `sudo`. 
+
+An example of building  docker image from Dockerfile adjusted to arm64:
```

Comment:
```suggestion
An example of building a docker image from a docker file adjusted to arm64:
```

Reply:
@Pezi777 The `Dockerfile` is a own name, used throughout Docker documentation as well as in Travis - I would leave it as it is for coherency?